integrationtest/wildfly/pom.xml

```diff
@@ -418,6 +420,75 @@
                             </dependency>
                         </dependencies>
                     </plugin>
+                    <plugin>
```

Comment:
I guess this works, but you could have asked the provisioning plugin to just replace the version during initial resolution?

Reply:
I wrote this (actually, you did and I copy/pasted for CDI) before we were even using feature packs :) I can replace it though.

Reply:
I just checked, that might not be easy. The <version-overrides> schema in [1] seems able to re-define the version, but not remap the other GAV elements to a different artifact altogether :(

1 - https://github.com/wildfly/wildfly-build-tools/blob/master/provisioning/src/main/resources/wildfly-server-provisioning-1_2.xsd